server.js

```diff
@@ -1,14 +1,40 @@
+/*eslint-env node*/
 
-// make this a simple http server that writes to the response stream object
+//http and server
+var http = require('http');
 
-// the page should display the following message:
+//trucks
+var foodTrucks = require('./trucks');
 
-// Today is <day name>, <month name> <date>. Here are the available food trucks:
+//date
+var theDate = require('./enhanced-date');
+theDate.setDate();
```

Comment:
I would move this into the handler function you pass to createServer. Once your server starts, it could theoretically run for days, meaning after the first day, anyone visiting the site would get results for just that first day you started the server.